src/IROperator.h

```diff
@@ -820,7 +885,12 @@ inline Expr pow(Expr x, Expr y) {
     if (x.type() == Float(64)) {
         y = cast<double>(y);
         return Internal::Call::make(Float(64), "pow_f64", {x, y}, Internal::Call::Extern);
-    } else {
+    }
+    else if (x.type() == Float(16)) {
+         y = cast<float16_t>(y);
+        return Internal::Call::make(Float(16), "pow_f16", {x, y}, Internal::Call::Extern);
```

Comment:
They don't. The commit message comments on this.

My idea here is that I don't want arguments to these math functions to be implicitly casted to float32 if they are of type float16. No one is using float16 expressions yet so this won't break any existing uses. As I experiment with backends I can add support as I go as needed.